common/config/rush/pnpm-lock.yaml

```diff
@@ -380,7 +380,7 @@ packages:
       integrity: sha512-CxaMaEjwtsmIhWtjHyGimKO7RmES0YxPqGQ9+jKqGygNlhG5NYHktDaiQu6w7k3g+I51VaLXtVSt+BVFd6VWfQ==
   /@azure/identity/1.2.5_debug@4.3.2:
     dependencies:
-      '@azure/core-http': 1.2.6
+      '@azure/core-http': 1.2.3
```

Comment:
why downgraded?

Reply:
Don't know either. I didn't change anything related to core-http.